ingest-app/resources/bulk_update_schema.json

```diff
@@ -12,6 +12,10 @@
         "UpdateTypeEnum": {
             "type": "string",
             "enum": ["ADD_TO_EXISTING", "CLEAR_FIELD", "CLEAR_ALL_AND_REPLACE", "FIND_AND_REMOVE", "FIND_AND_REPLACE"]
+        },
+        "UpdateFieldEnum": {
+          "type": "string",
+          "enum": ["SCIENCE_KEYWORDS", "LOCATION_KEYWORDS", "DATA_CENTERS", "PLATFORMS", "INSTRUMENTS"]
```

Comment:
I added them, but from a code coverage perspective everything is covered. It's using all the same functions and I am making sure it's all covered.

As for the API documentation, that is a separate ticket. I have not documented the API at this point because right now I don't want anyone (besides the MMT team) using bulk update. I have a ticket to update next sprint after I do the performance testing.